indico/modules/events/papers/templates/management/cfp_actions/_base.html

```diff
@@ -0,0 +1,27 @@
+{% extends '_module_actions_base.html' %}
+
+{% block icon_classes %}icon-file{% endblock %}
+
+{% block buttons %}
+    {% set title -%}
+        {%- if cfp.start_dt -%}
+            {%- trans %}Reschedule call for papers{% endtrans -%}
```

Comment:
You don't need both `-%}` and `{%-`. One of them will do the job.
If you have:
```
{% foo -%}
    {% bar %}
```
It's the same as having
```
{% foo %}
    {%- bar %}
```
Both of the remove the same empty space:
```
{% foo %}xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
xxxx{% bar %}
```